### PR TITLE
Add slug and tags to config.json

### LINKED ISSUE #117: [v3] Add key features
_This issue is part of the migration to v3. You can read full details about the various changes [here](https://github.com/exercism/v3-launch/)._

In Exercism v3, each track must specify exactly six "key features". Exercism uses these features to highlight the most interesting, unique or "best" features of a language to a student.

Key features are specified in the top-level `"key_features"` field in the track's `config.json` file and are defined as an array of objects, as specified in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Goal

The `"key_features"` field in the `config.json` file should be updated to describe the six "key features" of this track. See [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Example

```json
{
  "key_features": [
    {
      "icon": "features-oop",
      "title": "Modern",
      "content": "C# is a modern, fast-evolving language."
    },
    {
      "icon": "features-strongly-typed",
      "title": "Cross-platform",
      "content": "C# runs on almost any platform and chipset."
    },
    {
      "icon": "features-functional",
      "title": "Multi-paradigm",
      "content": "C# is primarily an object-oriented language, but also has lots of functional features."
    },
    {
      "icon": "features-lazy",
      "title": "General purpose",
      "content": "C# can be used for a wide variety of workloads, like websites, console applications, and even games."
    },
    {
      "icon": "features-declarative",
      "title": "Tooling",
      "content": "C# has excellent tooling, with linting and advanced refactoring options built-in."
    },
    {
      "icon": "features-generic",
      "title": "Documentation",
      "content": "Documentation is excellent and exhaustive, making it easy to get started with C#."
    }
  ]
}
```

## Tracking

https://github.com/exercism/v3-launch/issues/5

### LINKED ISSUE #118: [v3] Add tags
_This issue is part of the migration to v3. You can read full details about the various changes [here](https://github.com/exercism/v3-launch/)._

In Exercism v3, tracks can be annotated with tags. This allows searching for tracks with a certain tag combination, making it easy for students to find an interesting track to join.

Tags are specified in the top-level `"tags"` field in the track's `config.json` file and are defined as an array of strings, as specified in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#tags).

## Goal

The `"tags"` field in the `config.json` file should be updated to contain the tags that are relevant to this track. The list of tags that can be used is listed in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#tags).

## Example

```json
{
  "tags": [
    "runtime/jvm",
    "platform/windows",
    "platform/linux",
    "paradigm/declarative",
    "paradigm/functional",
    "paradigm/object_oriented"
  ]
}
```

## Tracking

https://github.com/exercism/v3-launch/issues/1

### LINKED ISSUE #137: Add language blurb
The track's `config.json` file must have a non-empty `blurb` property. This property should contain a short description (less than or equal to 400 characters) of the language. See https://github.com/exercism/docs/blob/main/building/tracks/config-json.md.

## Tracking

https://github.com/exercism/v3-launch/issues/37

### PR DESCRIPTION
**Pull request by 0x647262**
~~Closes #137~~
Closes #118 
Closes #117 

Filled the tags out to the best of my ability. If I'm missing anything, let me know and I'll get it added!

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "language": "D",
+  "blurb": "D is a general-purpose programming language with static typing, systems-level access, and C-like syntax. With the D Programming Language, write fast, read fast, and run fast.",
```
Comment (ErikSchierboom) [inline config.json:3]: I think this is the language's official introduction text verbatim, right? We usually don't want to copy things literally due to licensing issues. In https://github.com/exercism/d/pull/144/files, I took a stab at writing a blurb. How does that look like? Maybe you can use that and tweak it a bit?

**Comment by ErikSchierboom** [comment]
Thanks for working on this BTW!

File: config.json (hunk 1 above)
Comment (0x647262) [reply config.json]: I opted to leave your blurb as-is. I think it fits the language just fine!

**Comment by 0x647262** [comment]
@ErikSchierboom I'd like to tackle #117 with this merge as well, if possible.

I took a look at the key features documentation on: https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#key-features, but it seems there are no predefined features there. Would grabbing shared key features from other tracks `config.json` files be fine in this case?

**Comment by ErikSchierboom** [comment]
> [...] it seems there are no predefined features there.

That's correct. This is due to the key features being track-specific highlights.

> Would grabbing shared key features from other tracks config.json files be fine in this case?

The idea is that these key features showcase why a student would want to join that track. So usually you'd come up with the six things that showcase the track best, and then write the text for them. There is likely some overlap, so you could definitely take inspiration from other tracks, although it is likely that the text needs to be modified.

**Comment by 0x647262** [comment]
@ErikSchierboom all set!

**Review by ErikSchierboom — APPROVED** [review]
LGTM! Thanks

**Comment by ErikSchierboom** [comment]
@0x647262 You can see the key features on https://exercism.lol/tracks/d (provided you've not joined the track)

**Pull request merged**